packages/legend-studio/src/components/editor/edit-panel/mapping-editor/OperationSetImplementationEditor.tsx

```diff
@@ -104,12 +104,15 @@ export const OperationSetImplementationEditor = observer(
           );
         }
       };
+    const supportsParameters =
```

Comment:
Can we just flatten this, I don't think we need to unnecessarily trigger `handleDrop` recompute too many times